purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -490,7 +489,7 @@ internal class BillingWrapper(
         )
     }
 
-    @VisibleForTesting(otherwise = PRIVATE)
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
This broke compilation after the update for some reason... In any case, this is how we do it everywhere else.